.github/workflows/check.yml

```diff
@@ -10,6 +10,6 @@ jobs:
         runs-on: ubuntu-latest
         steps:
             - name: Checkout
-              uses: actions/checkout@v3
+              uses: actions/checkout@v4
```

Comment:
Should this be in this PR? shouldn't it be done in https://github.com/opennextjs/docs/pull/63 maybe?

Reply:
Good catch